packages/react/src/components/Switch/Switch.js

```diff
@@ -9,10 +9,11 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 import { settings } from 'carbon-components';
+import { matches, keyCodes } from '../../tools/key';
 
 const { prefix } = settings;
 
-const Switch = props => {
+const Switch = React.forwardRef(function Switch(props, tabRef) {
```

Comment:
Would we want to set `displayName` for Switch with this change?

Reply:
Sorry, I'm not sure what you mean by this. I know I'm missing but neither Switch or ContentSwitcher have a `displayName` prop? 😕